### PR TITLE
Modifications to the schema for validation errors

### PR DESCRIPTION
**Pull request by akaat**
The data failed to validate in Sage's system. Two types of changes are necessary - one is the format for the `scores` object (which does not require rebuilding the app) and the other is renaming the field that links to the schema. I suggest `schemaIdentifierPath` but if you have other suggestions, please make them and I will change the schema.

### CODE REVIEW

File: 3DRotation/3DRotationJSONSchema.json [hunk 1]
```diff
@@ -254,11 +254,12 @@
       "description" : "An identifier that can be used either by the developer or researcher for custom mapping.",
       "type" : "string"
     },
+    "schemaIdentifierPath" : {
+      "description" : "URL linking to the version of the schema against which these data should validate.",
+      "type" : "string"
+    },
```
Comment (syoung-smallwisdom) [inline 3DRotation/3DRotationJSONSchema.json:260]: @akaat This won't work. You are just introducing a *different* key that is not in the encoding.

File: 3DRotation/3DRotationJSONSchema.json [hunk 2]
```diff
@@ -254,11 +254,12 @@
       "description" : "An identifier that can be used either by the developer or researcher for custom mapping.",
       "type" : "string"
     },
+    "schemaIdentifierPath" : {
```
Comment (syoung-smallwisdom) [inline 3DRotation/3DRotationJSONSchema.json:257]:
```suggestion
    "$schema" : {
```

File: LetterNumberSeries/LetterNumberSeriesJSONSchema.json [hunk 3]
```diff
@@ -204,11 +204,12 @@
       "description" : "An identifier that can be used either by the developer or researcher for custom mapping.",
       "type" : "string"
     },
+    "schemaIdentifierPath" : {
```
Comment (syoung-smallwisdom) [inline LetterNumberSeries/LetterNumberSeriesJSONSchema.json:207]:
```suggestion
    "$schema" : {
```

File: ProgressiveMatrices/ProgressiveMatricesJSONSchema.json [hunk 4]
```diff
@@ -254,11 +254,12 @@
       "description" : "An identifier that can be used either by the developer or researcher for custom mapping.",
       "type" : "string"
     },
+    "schemaIdentifierPath" : {
```
Comment (syoung-smallwisdom) [inline ProgressiveMatrices/ProgressiveMatricesJSONSchema.json:257]:
```suggestion
    "$schema" : {
```

File: VerbalReasoning/VerbalReasoningJSONSchema.json [hunk 5]
```diff
@@ -204,11 +204,12 @@
       "description" : "An identifier that can be used either by the developer or researcher for custom mapping.",
       "type" : "string"
     },
+    "schemaIdentifierPath" : {
```
Comment (syoung-smallwisdom) [inline VerbalReasoning/VerbalReasoningJSONSchema.json:207]:
```suggestion
    "$schema" : {
```

**Review by syoung-smallwisdom — CHANGES_REQUESTED** [review]

**Comment by syoung-smallwisdom** [comment]
@akaat Both these changes require rebuilding the app b/c the URL includes the release tag.

**Comment by akaat** [comment]
Yes, it would require a rebuild. But it's bad practice to use the same name
for 2 different things even if they're in different spots.  We should be
rebuilding it. Or if we're not, it should only be for the pilot as a
band-aid.

**Review by syoung-smallwisdom — APPROVED** [review]

**Review by philerooski — COMMENTED** [review]
Looks good. Will `jsonSchema` be included in the data going forward? I can add it to the Glue table schemas in Bridge Downstream.

**Comment by akaat** [comment]
It is not included yet, but it is the most likely name for future data. If
you include it now, we will fix on that.

**Pull request merged**